scripts/syncSettings.py

```diff
@@ -58,6 +58,16 @@
         "url": "https://rpc.exosama.com",
         "blockReduced": 8192,
         "multiplierRequirement": 10000
+    },
+    "joc-mainnet": {
+        "url": "https://rpc-1.japanopenchain.org",
```

Comment:
Need to wait for JOC for a rpc information - these are not working so we either can't handle properly the snap/fast sync (would be very long longer-term) or we need to wait for a updated rpc.